README.md

```diff
@@ -451,6 +451,37 @@ You can decorate your own response headers by following the below example:
 Note: You need to specify `type` property when you decorate the response headers, otherwise the schema will be modified by Fastify.
 
 <a name="route.response.empty_body"></a>
+##### Different content types responses
+**Note:** not supported by Swagger (OpenAPI v2), [only OpenAPI v3](https://swagger.io/docs/specification/describing-responses/)
+<br>
+<br>
+Different content types responses are supported by `@fastify/swagger` and `@fastify`.
+Please use `content` for the response otherwise Fastify itself will fail to compile the schema:
```

Comment:
```suggestion
Use `content` for the response otherwise Fastify itself will fail to compile the schema:
```

Nothing to please here ;)

Reply:
`Please` is used everywhere in the readme